wikidata/entity.py

```diff
@@ -17,7 +17,7 @@
 if TYPE_CHECKING:
     from .client import Client  # noqa: F401
 
-__all__ = 'Entity', 'EntityId', 'EntityType'
+__all__ = ['Entity', 'EntityId', 'EntityState', 'EntityType']
```

Comment:
Please leave it as a `tuple`.

```suggestion
__all__ = 'Entity', 'EntityId', 'EntityState', 'EntityType'
```